content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: rolldice
+weight: 10
+---
+
+This page will show you how to get started with OpenTelemetry in C++.
+
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
+
+## Prerequisites 
+
+Ensure that you have the following installed locally:
+
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are 
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web frameworks as well.
+
+## Dependencies 
+
+**To begin, we will install Oatpp locally using the [source code](https://github.com/oatpp) and `make`, following these general steps:**
+
+1. Download oatpp source code:
+
+Obtain the oatpp source code. You can either download the source code from the oatpp Github repository or clone it using Git.
+
+```bash 
+git clone https://github.com/oatpp/oatpp.git
+```
+
+2. Navigate to the oatpp directory:
+
+Access the oatpp directory in your terminal.
+
+```bash 
+cd oatpp
+```
+
+3. Build oatpp:
+
+Build oatpp using the `make` command. This command will trigger the build process specified in the Makefile included in the oatpp source code.
+
+```bash
+make
+```
+4. Install oatpp:
+
+This command will install the built oatpp library and headers on your system, making it accesible for development in your project. 
+
+```bash
+make install
+```
+
+**Next, install and build [OpenTelemetry C++](https://github.com/open-telemetry/opentelemetry-cpp) locally using CMake, following these general steps:**
+
+1. Clone the OpenTelemetry C++ repository:
+
+Clone the OpenTelemetry C++ GitHub repository to your local machine.
+
+```bash
+git clone https://github.com/open-telemetry/opentelemetry-cpp.git
+```
+
+2. Navigate to the OpenTelemetry C++ SDK directory:
+
+Change your working directory to the OpenTelemetry C++ SDK directory.
+
+```bash
+cd openTelemetry-cpp
+```
+
+3. Create a build directory:
+
+It's a good practice to create a seperate directory for the build files to keep your source directory clean.
+
+```bash
+mkdir build
+cd build 
+```
+
+4. Configure and generate the build system:
+
+Run the CMake from the build directory to configure the build.
+
+```bash
+cmake ..
```

Comment:
This failed for me on MacOS, I needed to run

```bash
cmake -DWITH_ABSEIL=ON ..
```

On which OS did you run? We may need to provide options per OS here (I can help with that)

Reply:
interesting... it did work for me on MacOS. Yes, a lot of these commands are tailored for MacOS users - should I include for other OSs?

Reply:
We can worry about different OSes later, the PR is complex enough, let's make it fly for _some_ people & OSes and then rapidly following up with fixes.

Reply:
sounds good. So going back to the configure command ` cmake ..`  it didn't work for you and it might not work for others too. Would you like me to add both  `cmake -DWITH_ABSEIL=ON ..` and `cmake ..` in this pr

Reply:
yeah, let's do it that way! thanks!